cli/src/cli.mjs

```diff
@@ -470,18 +476,49 @@ if ( parsed.help ) {
     });    
   }
 
-}
-  else if (parsed.getStatus){
-    axios.get(url(host, port, '/api/v1/status'), undefined)
-      .then(function (response) {
-        console.log(response.data);
-      })
-      .catch(function (error) {
-        logError(error);
+} else if (parsed.getStatus) {
+  axios.get(url(host, port, '/api/v1/status'), undefined)
+    .then(function (response) {
+      console.log(response.data);
+    })
+    .catch(function (error) {
+      logError(error);
     });
-  }
-  else {
-
+} else if (parsed.downloadOverrides) {
+  /* overrideDirectory is mock-firebolt/cli/externalOverrides by default, 
+  if overrided via CLI, will have user given location to save repository contents */
+  const overrideDirectory = parsed.overrideLocation ? parsed.overrideLocation : '../externalOverrides'
+  // git repository url to be downloaded
+  const downloadUrl = parsed.downloadOverrides;
+  // Downloading git repository, creates a clone folder with repository name, inside current working directory.
+  let urlArray = downloadUrl.split(".git");
+  urlArray.pop()
+  let cloneFolder = (urlArray[urlArray.length - 1].split("/")).pop()
```

Comment:
If we're just gonna delete the cloneFolder afterwords let's just use something like "tmp" to avoid unnecessarily complex code.